src/components/ToolButton.tsx

```diff
@@ -41,7 +41,7 @@ type ToolButtonProps =
       onChange?(): void;
     });
 
-const DEFAULT_SIZE: ToolIconSize = "m";
+const DEFAULT_SIZE: ToolButtonSize = "medium";
```

Comment:
lets add this in default props and remove the or condition in the component

Reply:
I think it's fine kept in the component as it doesn't affect anything not is reused elsewhere. I believe it was factored into a constant just for type-safety.

Reply:
since this is one of the prop and we are setting default value for this prop, hence it will be better to set in default props instead, that way it will also be available in devtools as well

Reply:
Didn't even realize we're using those in this component. Not a fan of `defaultProps` coz it doesn't work well with TS. Alas, since we're already using it, let's do it.